conf/machine/apalis-imx6.conf

```diff
@@ -14,7 +14,7 @@ KERNEL_DEVICETREE += "imx6q-apalis-eval.dtb imx6q-apalis-ixora.dtb \
                       imx6q-apalis-ixora-v1.1.dtb"
 KERNEL_IMAGETYPE = "zImage"
 # The kernel lives in a seperate FAT partition, don't deploy it in /boot/
-RDEPENDS_${KERNEL_PACKAGE_NAME}-base = ""
+RDEPENDS:${KERNEL_PACKAGE_NAME}-base = ""
```

Comment:
Should stay as `RDEPENDS_`, right?

Reply:
Hmm, I see that `poky` has this as well. Not sure how this is different from `PREFERRED_PROVIDER_`...

Reply:
RDEPENDS_${PN} is override so the change is fine here.

Reply:
Sure, I'm not so troubled by that since at least it is consistent. But what makes `PREFERRED_PROVIDER_recipe` _not_ an override? Will the average user understand why there is a different character used here?

Reply:
yeah they are different semantically, PREFERRED_PROVIDER_<recipe> is a single variable.

Reply:
Unfortunately there are a number of acknowledged special cases with respect to what is an override and what is simply a variable name with and underscore and lower-case afterwards. There are also cases of overrides that don't appear in the OVERRIDES set. It's a bit of voodoo, and only bitbake itself is the final arbiter:

https://git.yoctoproject.org/cgit/cgit.cgi/yocto-docs/tree/documentation/migration-guides/migration-3.4.rst